src/toast/ops/conviqt.py

```diff
@@ -805,30 +851,35 @@ def _exec(self, data, detectors=None, **kwargs):
 
             detector = self.get_detector(det)
 
-            theta, phi, psi, psi_pol, hwp_angle = self.get_pointing(data, det, verbose)
+            theta, phi, psi_det, psi_pol, psi_beam, hwp_angle = self.get_pointing(
+                data, det, verbose
+            )
+
             # T-convolution
-            pnt = self.get_buffer(theta, phi, psi, det, verbose)
+            pnt = self.get_buffer(theta, phi, psi_beam, det, verbose)
 
             convolved_data = self.convolve(
                 skyT, beam_T, detector, pnt, det, verbose, pol=False
             )
 
             if self.pol:
                 del (pnt,)
+                angle_arg = 4.0 * hwp_angle
```

Comment:
We shouldn't multiply here by 4 , should we ? see above https://github.com/hpc4cmb/toast/pull/663/files#r1221116956